README.md

```diff
@@ -53,4 +55,34 @@ look like this:
     |-- <QBIC sample code>.fastq.gz.sha256sum
 ```
 
+### HLA Typing data
+**Responsible dropbox:**
+[QBiC-register-hlatyping-dropbox](drop-boxes/register-hlatyping-dropbox)
+
+**Resulting data model in openBIS**  
+Q_TEST_SAMPLE -> Q_NGS_HLATYPING (with sample code) -> DataSet (directory
+with files contained)
+
+or
+
+Q_TEST_SAMPLE -> Q_NGS_SINGLE_SAMPLE_RUN (provided sample code) -> Q_NGS_HLATYPING -> DataSet (directory
+with files contained)
```

Comment:
I would like to know and have it documented, why there are to models resulting from one structure in one ETL :)

Reply:
This was also something @wow-such-code pointed out. One is a special case that describes how a specific customer registers his data. But as I understood ideally it should be done like this:
```
Q_TEST_SAMPLE -> Q_NGS_SINGLE_SAMPLE_RUN (provided sample code) -> Q_NGS_HLATYPING -> DataSet (directory
with files contained)
```

Reply:
ok, beside that is really bad, we need to document this switch. We get rid of this branching when we work on the harmonisation of etls.

Reply:
Can you also create an issue for this please?

Reply:
These are the two cases that I found in production, the unorthodox one (hla-typing attached to test sample) is from 2015 though, so it might have been a problem only early on. maybe @christopher-mohr knows more

Reply:
@wow-such-code but what does the HLA ETL reflect at the moment?

Reply:
we definitely want the second scenario, but there are no tests that exclude the first